### DISCUSSION TITLE
Only (Avalonia UI) options are available when creating a new Item

### DISCUSSION THREAD

**Discussion by Chesno4ok** [AvaloniaUI/Avalonia#16692] [Q&A] [answered]
I used to create new elements using (Avalonia) elements, like those below:
![image](https://github.com/user-attachments/assets/f6d010a5-5a74-43cd-866a-8f9344a9f319)

But recently I tried to create a new one and the only ones available are (Avalonia UI) options, and the other ones disappeared.
![image](https://github.com/user-attachments/assets/b406f121-3cd2-4528-b42e-58015f0587b5)

Were they deleted with a new update or is this some sort of a bug?

**Comment by stevemonaco** [comment, marked as answer]
Avalonia was previously providing two sets of templates: one via AvaloniaVS and one via avalonia-dotnet-templates. The AvaloniaVS templates were removed in https://github.com/AvaloniaUI/AvaloniaVS/pull/453

**Reply by Chesno4ok** [reply]
Is there a way to bring them back?

**Reply by stevemonaco** [reply]
No, they've been removed so the team won't be required to maintain two sets of templates.

Is there a reason why you want to bring them back? The only core difference, AFAIK, is the the VS ones matched namespaces to folder hierarchy automatically. The avalonia-dotnet-template ones currently can't because of a bug/limitation in the .NET template engine that hasn't been addressed yet.

**Reply by Chesno4ok** [reply]
Turns out Templated Controls in Avalonia VS were made using Styles, when in the avalonia-dotnet-templates it uses ResourceDictionary. It didn't work at first and I thought I had to connect it in App.axaml, but turned out I just had to adjust namespace to match the folder (as you've mentioned earlier). Everything works now, thanks for your help!
